### ISSUE TITLE
AndroidX?

### ISSUE THREAD

**Issue by JoaquimLey** [googlesamples/android-sunflower#2]
Hello, I was wondering if you guys/gals are going to do a namespace migration to `androidx` in this project.

I'm asking this because on my current project I'm getting errors:

```groovy
Failed to resolve: com.google.android.material:material:1.1.0
Failed to resolve: androidx.fragment:fragment:1.1.0
Failed to resolve: androidx.recyclerview:recyclerview:1.1.0

(...)
```

It would be good to see a working sample with recommended practices. I've been following the guide but with no success, also I included the experimental:

```groovy
# AndroidX
android.enableJetifier=true
android.useAndroidX=true
```

Running on:

```
Android Studio 3.2 Canary 15
Build #AI-181.4668.68.32.4773949, built on May 10, 2018
JRE: 1.8.0_152-release-1136-b02 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.12.6
```

----------------------------------

**Project:** https://github.com/JoaquimLey/transport-eta/tree/migration/jetpack

**Comment by JoaquimLey** [comment]
Just a follow-up, there is already a project with `androidx` dependencies namespace here:

- https://github.com/googlesamples/android-SliceViewer

It is compiling on my machine, but still on the above mentioned project I can't, maybe I'm missing some configuration (haven't dived deep), just an FYI if someone stumbles upon this.

**Comment by lianyueforever** [comment]
[https://developer.android.com/topic/libraries/support-library/androidx-rn](url)
just 1.0.0-alpha2

**Comment by JoaquimLey** [comment]
@lianyueforever thanks for the update, can you please elaborate?

**Comment by JoaquimLey** [comment]
I was able to Migrate after removing all the dependencies, I guess this was a cold cache refresh (I had tried to remove the `.gradle` and invalidate caches but no luck anyway here's the commit with the changes if anyone is interested:

- https://github.com/JoaquimLey/transport-eta/commit/a3508fb9601a01f9cf7a931c5d24d72599680da8

------------------------------------

_P.S: I'm leaving this issue open since there hasn't been any response from the contributors' team, and I am still interested in seeing a Google migration._

**Comment by tiembo** [comment]
Yes, there will be a migration to AndroidX.  Currently Navigation + AndroidX is only possible with Android Gradle Plugin 3.2.0-alpha17, which will be released soon.  Once that lands work will resume on migrating.

See https://issuetracker.google.com/issues/79667498 for more details.

**Comment by JoaquimLey** [comment]
Awesome, thanks for the feedback @tiembo, feel free to close this issue.
FYI I haven't tried the navigation but was able to get it working.

**Comment by tiembo** [comment]
Reopening and assigning to me, as we do want to incorporate AndroidX

**Comment by tiembo** [comment]
Migrating to AndroidX is still on hold due to https://issuetracker.google.com/issues/110692942.

**Comment by iNoles** [comment]
@tiembo I am still waiting for https://issuetracker.google.com/issues/112105577 too.

**Comment by tiembo** [comment]
Resolved by #185
